internal/guest/runtime/hcsv2/workload_container.go

```diff
@@ -119,6 +121,22 @@ func setupWorkloadContainerSpec(ctx context.Context, sbid, id string, spec *oci.
 	// also has a concept of a sandbox/shm file when the IPC NamespaceMode !=
 	// NODE.
 
+	if val, ok := spec.Annotations["io.microsoft.container.storage.shm.size-kb"]; ok {
```

Comment:
What happens if someone specifies 0?

Reply:
I tried this, so turns out nothing will be mounted. maybe it's worth adding a check

Reply:
Sounds like a good idea